### PR TITLE
ember-cli@2.10.0

### PR DESCRIPTION
**Pull request by kellyselden**

### CODE REVIEW

**Comment by kellyselden** [comment]
https://github.com/ember-cli/ember-cli/pull/6496 stops this

File: package.json [hunk 1]
```diff
@@ -2,54 +2,54 @@
...
     "ember-cli-inject-live-reload": "^1.4.1",
-    "ember-cli-jshint": "^2.0.0",
+    "ember-cli-jshint": "^2.0.1",
     "ember-cli-qunit": "^3.0.1",
     "ember-cli-release": "^0.2.9",
     "ember-cli-sri": "^2.1.0",
     "ember-cli-test-loader": "^1.1.0",
     "ember-cli-uglify": "^1.2.0",
-    "ember-data": "^2.9.0",
+    "ember-data": "^2.10.0",
```
Comment (mriska) [inline package.json]: Question: Do you actually use `ember-data` in this addon or could this dependency be removed?

Comment (kellyselden) [inline package.json]: Yes, some of the promise macros use mixins from it. But I'm totally open to extracting those macros out to remove the ember-data dependency.

Comment (mriska) [inline package.json]: Ah, ok. just thought it might be totally unused and just part of the basic addon scaffolding.

Comment (kellyselden) [inline package.json]: added tracking issue #102

Comment (mriska) [inline package.json]: 102 or 161?

Comment (kellyselden) [inline package.json]: you're right, #161

**Pull request merged**